core/commands/dag/dag.go

```diff
@@ -109,7 +109,7 @@ format.
 		cmds.StringArg("ref", true, false, "The object to get").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("format", "f", "Format that the object will be serialized as.").WithDefault("dag-json"),
+		cmds.StringOption("codec", "c", "Format that the object will be encoded as.").WithDefault("dag-json"),
```

Comment:
"c" is used for the global `--config`. We can either not use a shortcut or come up with a different name/leave things as is.

Reply:
I don't think short notation adds value here, I'm for just removing it (see https://github.com/ipfs/go-ipfs/pull/8439#discussion_r715076778)

Reply:
Going to remove the shortcut and change to `--output-codec` to clarify that the stored data is being transcoded per https://github.com/ipfs/go-ipfs/pull/8439#issuecomment-928151167.